kbs/config/kubernetes/deploy-kbs.sh

```diff
@@ -31,6 +33,15 @@ kbs_cert="${k8s_cnf_dir}/base/kbs.pem"
     openssl pkey -in "${k8s_cnf_dir}/base/kbs.key" -pubout -out "${kbs_cert}"
 }
 
+# Enable the nvidia remote verifier if requested.
+if [[ "${NVIDIA_VERIFIER_MODE}" == "remote" ]]; then
+    cat >> ${k8s_cnf_dir}/base/kbs-config.toml << 'EOF'
+
+[attestation_service.verifier_config.nvidia_verifier]
+type = "Remote"
+
+EOF
+fi
```

Comment:
Can we change the original config file than modify it in a script here? It could help avoid maintain config file in two places.

Reply:
I was also thinking that updating the `.toml` file directly should be ok

Reply:
Hmm you mean just always have the remote verifier enabled in K8S? That might be fine for the CI, but this script is also for general users to deploy Trustee in k8s. The remote verifier has some licensing implications so it might be best to require them to opt-in.

Also note that https://github.com/kata-containers/kata-containers/pull/12041 was already merged.

It would simplify things to make it always enabled tho. Wdyt?